distro/adaptation/amazon_linux

```diff
@@ -18,4 +18,8 @@ default-jdk: java-17-amazon-corretto
 dnsutils: bind-utils
 dpkg-dev:
 e2fslibs-dev: e2fsprogs-devel
-exfat-fuse:  
+exfat-fuse: 
+f2fs-tools: 
+fakeroot:
+freeglut3: freeglut
```

Comment:
take this as example, since it is common mapping, this commit should also remove it from amazon_linux-2 and amazon_linux-2022